app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,10 @@
+<% provide(:title, t(".title")) %>
+<h1><%= t(".about_page")%></h1>
+<p>
+   <%= t(".content_massage", link_home: link_to(t(".rubyonrails_href"), "https://www.railstutorial.org/"),
+                    link_book: link_to(t(".book_href"), "https://www.railstutorial.org/book/"),
+                    link_screencast: link_to(t(".screencasts_href"),
+                                              "http://screencasts.railstutorial.org/"),
+                    link_rails: link_to(t(".rails_href"), "http://rubyonrails.org/")
+        ).html_safe %>
```

Comment:
indent sai nhé

`html_safe` - giải thích code này dùng để làm gì em

Reply:
>`html_safe - giải thích code này dùng để làm gì em`
nội dung này em chưa giải thích nè

Reply:
Dạ trong trường hợp này html_safe là để trả lại các chuỗi HTML. Nếu không có html_safe thì khi sử dụng link_to để link đến các địa chỉ thì nó sẽ in ra thẻ html của địa chỉ đó và không trỏ đến địa chỉ đó được